regex/rstring/string_test.go

```diff
@@ -77,3 +77,15 @@ func ExampleSplitString_other() {
 	fmt.Println(rstring.SplitString(-1, "mango     is     fruit"))
 	// Output: [mango is fruit]
 }
+
+func TestRemoveLineBreaks(t *testing.T) {
+	input := "One\r,\r\ntwo\u0085 three\u2028!\u2029'"
+	result := "One,two three!'"
+	output := rstring.RemoveLineBreaks(input)
+	diff := reflect.DeepEqual(result, output)
+
```

Comment:
@binkkatal please modify method such that we can test for multiple inputs with ease, check other methods in this file

Reply:
@patil-ashutosh , added subtest , ready for re-review